libs/community/langchain_community/chat_models/openai.py

```diff
@@ -143,6 +144,7 @@ def _convert_delta_to_message_chunk(
         return default_class(content=content)
 
 
+@deprecated(since="0.1.0", removal="0.2.0", alternative="langchain_openai.ChatOpenAI")
```

Comment:
causes a decent number of deprecation warnings in unit tests, but still important to test community one before removal.